site/docs/v1/tech/account-management/index.adoc

```diff
@@ -36,9 +38,13 @@ Erlich Bachman for any `admin` facing actions. Imagine Erlich is you or a custom
 // box shadow to be removed during next style release
 image::account-management/erlich-bachman-small-profile.png[User Profile Image Erlich, width=50, role=box-shadow]
 
+==== Defaults Shown in Screenshots
+
+Additionally, all screenshots shown are reflective of the default themes and forms shipped with FusionAuth. In a production environment, themed pages and forms will likely differ according to your brand and product requirements.
```

Comment:
"all screenshots shown are reflective of the default themes and" => "all screenshots show the default theme and"